dashboard/app/views/levels/_multi.html.haml

```diff
@@ -71,259 +83,21 @@
 
 = render partial: 'levels/common_audio'
 
+- # Only include multi.js once per page
+- if !@included_multi
```

Comment:
Hooray for pulling JavaScript out of our views!

It's true I'd prefer to put levels/multi.js in the code-studio package - see how [levels/contract_match.jsx and levels/widget.js get defined as build targets here](https://github.com/code-dot-org/code-dot-org/blob/staging/code-studio/scripts/build-js.js#L36) with their common code automatically factored out.

This is still a tricky situation; We sometimes include this partial once by itself, and sometimes several times from another partial - in both cases we only want one `<script>` tag?  Code-studio doesn't solve the problem at this layer, it's good for deduplicating dependencies within the package.  I'd actually extract another partial:
```
_level_group.haml
  levels.each do
    _single_multi.haml

_multi.html.haml
  _single_multi.haml
```
Then do the JavaScript include in the root partial for each case, preferably in a [`content_for(:head)` block](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/_dsl.html.haml#L1).

Not sure that's significantly better than what's happening here now, though.